### DISCUSSION TITLE
Wiki is accessible through hostnames that are not the canonical host

### DISCUSSION THREAD

**Discussion by Csardelacal** [requarks/wiki#4632] [Help / Questions]
Hey there!

I run wiki.js in a docker container (v2.5.219) and I seem to find no setting to disable accessing the wiki from either bad hostnames or the wiki's IP address. I've seen it as a common practice that in production accessing a website through the non-canonical address gets either a not found response or a redirection to canonical.

Is it possible to enable these kinds of redirections once the wiki has been properly set up?

**Comment by ieaster1** [comment]
I believe what you're asking is beyond the scope of the web application.  You should look to use nginx for your front access to give you redirect ability.

**Reply by Csardelacal** [reply]
I think you're answering my question and at  the same time not answering my question. I understand from your reply that wiki.js is not capable of adding a redirection if the request is being sent from a different host than the one it expects. Am I correct?

About the scope, wiki.js already handles SSL certificates and has a built in web server. It's not like it's a CGI or SAPI application that can't function without a web server. Why would it be unreasonable to expect it to verify the `host` header?

**Reply by ieaster1** [reply]
Yes, I see your point.  I'll step away for someone else more qualified to weigh in.  :)
